JSONModel/JSONModelTransformations/JSONKeyMapper.m

```diff
@@ -171,4 +171,45 @@ +(instancetype)mapperFromUpperCaseToLowerCase
 
 }
 
++ (instancetype)mapper:(JSONKeyMapper *)baseKeyMapper
+        withExceptions:(NSDictionary *)exceptions
+{
+    NSDictionary* userToModelMap = [exceptions copy];
+    NSDictionary* userToJSONMap  = [NSDictionary dictionaryWithObjects:exceptions.allKeys forKeys:exceptions.allValues];
```

Comment:
This is not an acceptable/safe way to flip the keys/values of a dictionary. The documentation for `allKeys` and `allValues` specifically states that the order is undefined.